### DISCUSSION TITLE
Direct acces to menu on widdget in upper bar

### DISCUSSION THREAD

**Discussion by MicheleVilla** [EdgeTX/edgetx#3003] [Ideas]
Hi
For touch display
it might be useful to have a widget in the top bar that allows direct access to a menu chosen by the user
in this way you avoid multiple steps to get to the requested menu

**Comment by offer-shmuely** [comment]
isn't that already working?

**Reply by pfeerick** [reply]
I don't think so... think he is effectively asking for a shortcut widget.
